deployment/docker/README.md

```diff
@@ -128,3 +128,13 @@ of alerting rules is the following:
 
 Please, also see [how to monitor](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#monitoring) 
 VictoriaMetrics installations.
+
+## VictoriaLogs server
```

Comment:
we have references for this Docker files, should we add the same one for VIctoriaLogs documentation?

Reply:
good point, added